lib/sneakers/queue.rb

```diff
@@ -16,7 +16,10 @@ def initialize(name, opts)
   # :ack
   #
   def subscribe(worker)
-    @bunny = Bunny.new(@opts[:amqp], :vhost => @opts[:vhost], :heartbeat => @opts[:heartbeat], :logger => Sneakers::logger)
+    # If we've already got a bunny object, use it.  This allows people to
+    # specify all kinds of options we don't need to know about (e.g. for ssl).
+    @bunny = @opts[:bunny]
+    @bunny ||= Bunny.new(@opts[:amqp], :vhost => @opts[:vhost], :heartbeat => @opts[:heartbeat], :logger => Sneakers::logger)
```

Comment:
One suggestion here: maybe create a private `create_bunny_connection` method, and extract the code 
```ruby
def create_bunny_connection
  Bunny.new(
    @opts[:amqp], 
    :vhost => @opts[:vhost], 
    :heartbeat => @opts[:heartbeat], 
    :logger => Sneakers::logger`
  )
end
```

into it. I mean, that way it's event more obvious what's happening